src/encoded/dev_servers.py

```diff
@@ -1,192 +1,3 @@
-
-    @atexit.register
-    def cleanup_process():
-        for process in processes:
-            if process.poll() is None:
-                process.terminate()
-        for process in processes:
-            try:
-                for line in process.stdout:
-                    sys.stdout.write(line.decode('utf-8'))
-            except IOError:
-                pass
-            process.wait()
-
-    postgres = postgresql_fixture.server_process(pgdata, echo=True)
-    processes.append(postgres)
-
-    es_server_url = config.get('elasticsearch.server', "localhost")
-
-    if ("127.0.0.1" in es_server_url or "localhost" in es_server_url):
-        # Bootup local ES server subprocess. Else assume connecting to remote ES cluster.
-        elasticsearch = elasticsearch_fixture.server_process(esdata, echo=True)
-        processes.append(elasticsearch)
-    elif not config.get('indexer.namespace'):
-        raise Exception("It looks like are connecting to remote elasticsearch.server but no indexer.namespace is defined.")
-    elif not config.get("elasticsearch.aws_auth", False):
-        # TODO detect if connecting to AWS or not before raising an Exception.
-        PRINT("WARNING - elasticsearch.aws_auth is set to false. Connection will fail if connecting to remote ES cluster on AWS.")
-
-    nginx = nginx_server_process(echo=True)
-    processes.append(nginx)
-
-    if ingest:
-        ingestion_listener = ingestion_listener_process(config_uri, app_name)
-        processes.append(ingestion_listener)
-
-    if init:
-        app = get_app(config_uri, app_name)
-    else:
-        app = None
-
-    # clear queues and initialize indices before loading data. No indexing yet.
-    # this is needed for items with properties stored in ES
-    if init:
-        create_mapping.run(app, skip_indexing=True, purge_queue=False)
-
-    if init and load:
-        load_test_data = app.registry.settings.get('load_test_data')
-        load_test_data = DottedNameResolver().resolve(load_test_data)
-        load_res = load_test_data(app)
-        if load_res:  # None if successful
-            raise load_res
-
-        # now clear the queues and queue items for indexing
-        create_mapping.run(app, check_first=True, strict=True, purge_queue=False)
-
-    PRINT('Started. ^C to exit.')
-
-    stdouts = [p.stdout for p in processes]
-
-    # Ugly should probably use threads instead
-    while True:
-        readable, writable, err = select.select(stdouts, [], stdouts, 5)
-        for stdout in readable:
-            for line in iter(stdout.readline, b''):
-                sys.stdout.write(line.decode('utf-8'))
-        if err:
-            for stdout in err:
-                for line in iter(stdout.readline, b''):
-                    sys.stdout.write(line.decode('utf-8'))
-            break
-
-
-if __name__ == '__main__':
-    main()
+from snovault import dev_servers
+from . import project_defs  # noqa: F401 (imported but unused)
+dev_servers.main()
```

Comment:
As with Fourfront, this needs to be 
```suggestion
if __name__ == "__main__":
    dev_servers.main()
```